test/jest/cookieSyncManager.spec.ts

```diff
@@ -13,10 +13,12 @@ const pixelSettings: IPixelConfiguration = {
     isProduction: true,
     settings: {},
     frequencyCap: 14,
-    pixelUrl: '',
-    redirectUrl: '',
+    pixelUrl: 'https://test.com',
+    redirectUrl: '?redirect=https://redirect.com&mpid=%%mpid%%',
```

Comment:
If we are populating this object with default values, we should have a test to handle the expected behavior if they are missing. I don't recall if I did that.